Test/JwtTest/JwtTokenGenerationTests.cs

```diff
@@ -0,0 +1,246 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Test.JwtTest
+{
+    public class JwtTokenGenerationTests
+    {
+        private readonly string _issuer;
+        private readonly string _audience;
+        private readonly string _secret;
+
+        public JwtTokenGenerationTests()
+        {
+            var configuration = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .Build();
```

Comment:
Kan se du laver noget data, men lige for at være sikker, er det noget mocked configuration eller noget "actual stuff"